drivers/wifi/esp_at/esp.c

```diff
@@ -509,6 +509,8 @@ MODEM_CMD_DEFINE(on_cmd_closed)
 
 	link_id = data->match_buf[0] - '0';
 
+	LOG_WRN("Link %d closed", link_id);
+
```

Comment:
I think that this flow can happen frequently in some user scenarios. One that comes to my mind is HTTP request (client connection from Zephyr point of view), where HTTP server will automatically close TCP connection after replying. In such case this socket will be closed "asynchronously", so warning will pop up on the console.

Additionally such event is propagated first to the ESP workqueue, then to the networking socket layer, so application developer should get notified about closed socket.

IMO this message should be DBG at most, not WRN, for the reason I explained.

Reply:
Good point, with MQTT sockets shouldn't ever be closed so I would prefer a warning, but obviously there are many normal situations when links are closed. Updated to DBG.

Reply:
>Updated to DBG.

@JordanYates Did you forget to push?

Reply:
I apparently updated the commit message without updating the code, fixed now.